src/Card/CardFooter.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import CardContext from './CardContext';
+
+const CardFooter = React.forwardRef(({
+  children,
+  className,
+  isStacked,
+  text,
+  orientation,
+}, ref) => {
+  const { orientation: contextOrientation } = useContext(CardContext);
+  const footerOrientation = orientation || contextOrientation;
+  const wrapperClassName = `pgn__card-footer ${footerOrientation}${isStacked ? '-stacked' : ''}`;
+  const textClassName = `pgn__card-footer-text ${footerOrientation}${isStacked ? '-stacked' : ''} x-small`;
+
+  return (
+    <div className={classNames(className, wrapperClassName)} ref={ref}>
+      {text && <div className={textClassName}>{text}</div>}
+      {children}
+    </div>
+  );
+});
+
+CardFooter.propTypes = {
+  /** Specifies contents of the component. */
+  children: PropTypes.node.isRequired,
+  /** Specifies class name to append to the base element. */
+  className: PropTypes.string,
+  /** Footer text to display near actions. */
+  text: PropTypes.string,
```

Comment:
I think we'll want to support passing any node and/or element here, rather than just a string. Looking at the Figma examples, looks like we might want to pass in other components into this place (e.g., "Related programs"):

![image](https://user-images.githubusercontent.com/2828721/146397751-6a104a74-cc6e-4fb6-91e2-b9c14f164927.png)

Reply:
Yeah, I've missed that example, It also makes sense that we should support nodes and not only strings there. Updated it, thank!